src/index.html

```diff
@@ -5,8 +5,432 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page">
+    <header class="header">
+      <a href="#home">
+        <img
+          src="images/logo.png"
+          alt="MOYOlogo"
+          class="header__icon"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link nav__link--is-active" href="#apple">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#samsung">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#phones">Смартфоны и телефоны</a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#computers"
+              data-qa="hover"
+            >
+              Ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#gadgets">Гаджеты</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#tablets">Планшеты</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#photos">Фото</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#videos">Видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="assortment">
+        <main class="assortment__card card" data-qa="card">
+          <img
+            src="images/imac.jpeg"
+            alt="iMac"
+            class="card__picture"
+          >
+          <section class="content">
+            <div class="content__tittle">
+              Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+            </div>
+
+            <div class="content__code">
+              Код товара: 195434
+            </div>
+
+            <div class="content__reviews reviews">
+              <div class="reviews__stars stars stars_stars--4">
+                <span class="star stars__star"></span>
+                <span class="star stars__star"></span>
+                <span class="star stars__star"></span>
+                <span class="star stars__star"></span>
+                <span class="star stars__star"></span>
```

Comment:
```suggestion
                <span class="star stars__star"></span>
                <span class="star stars__star"></span>
                <span class="star stars__star"></span>
                <span class="star stars__star"></span>
                <span class="star stars__star"></span>
```
do you need both classes?